installer/templates/phx_web/components/core_components.ex

```diff
@@ -106,10 +106,12 @@ defmodule <%= @web_namespace %>.CoreComponents do
   slot :inner_block, doc: "the optional inner block that renders the flash message"
 
   def flash(assigns) do
+    assigns = assign(assigns, :id, assigns.id || "flash-#{assigns.kind}")
```

Comment:
Can we use assign_new instead? :)

Reply:
This handles assigns has `id` with `nil` value - but `assign_new` would not set the value in that case since it checks whether the key exist or not.

I intentionally used `||` here to avoid having `nil` value for id in any cases.. but should I handle that here? Or should we change `assign_new` to handle that case? hmm..

Anyway, just let me know what you think - I'll update the PR accordingly.

Reply:
I think if you set `id={nil}` I would not expect that to work. As setting it to `id={{:inside_tuple}}`  won't work either.

Reply:
That makes sense - updated to use `assign_new`